packages/opentelemetry-plugin-express/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@opentelemetry/plugin-express",
+  "version": "0.3.2",
```

Comment:
```suggestion
  "version": "0.4.0",
```

Reply:
Any update here?

Reply:
I forgot to update this one my bad, i will update tomorrow

Reply:
Updated to 0.4.0 !